src/components/v2/AuthModal/WalletList/index.tsx

```diff
@@ -0,0 +1,69 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import Typography from '@mui/material/Typography';
+
+import { Connector } from 'clients/web3';
+import { VENUS_TERMS_OF_SERVICE_URL } from 'config';
+import { Icon } from '../../Icon';
+import { WALLETS, UPCOMING_WALLETS } from '../constants';
+import { useStyles } from './styles';
+
+export interface IWalletListProps {
+  onLogin: (connector: Connector) => void;
+}
+
+export const WalletList: React.FC<IWalletListProps> = ({ onLogin }) => {
+  const styles = useStyles();
+
+  return (
+    <div css={styles.container}>
+      {WALLETS.map(({ name, connector, Logo }) => (
+        <button
```

Comment:
What is missing that our button component misses this use case?

Reply:
Our button component would never cover this case. The `button` tag is used semantically here, this is what it looks like in the UI:
<img width="566" alt="Screen Shot 2022-03-28 at 4 47 59 PM" src="https://user-images.githubusercontent.com/44675210/160436941-1addfbae-3b13-4c3a-b7a1-4095bbf29b31.png">

Reply:
It could be written with the two icons and text positioned but I agree it would require additional styles. The advantage to using our button component is interactive styles would be consistently applied

Reply:
The problem is that in this case they don't share any styling, including the `hover` and `active` styles.
For other interactive styles, like the `outline` property, these should be defined globally for the whole project in my opinion.